index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__bg">
+        <img src="img/bg.png" alt="bg" class="header__bg-content" />
+      </div>
+      <div class="header__flex-container">
+        <div class="header__flex-container-logobox">My Logo</div>
+        <div class="header__flex-container-menu">
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="header__center">
+        <h1 class="heading__primary">I’m Isaac Glad to see you!</h1>
+        <div class="heading__primary-content">
```

Comment:
- [ ] h1 header on mobile view is too big, please make it match the figma design: 

Yours  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/153431244-771626b2-3a76-462a-b6ba-17fd3f839ccb.png)  | ![image](https://user-images.githubusercontent.com/77038610/153431313-05c81fec-383b-433d-bbf5-54141e0e4296.png)

- [ ] Please use colors provided in figma, you can find details in right menu (typography, colors sections), in case of `h1`, please use #172B4D color:

![image](https://user-images.githubusercontent.com/77038610/153432805-39da4f91-1c37-4ce9-a5d5-dc9a2fe4839b.png)